features/org.wso2.carbon.identity.organization.management.core.feature/resources/organization-mgt.xml.j2

```diff
@@ -0,0 +1,78 @@
+<!--
+~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
check the license header

Reply:
Sorry, missed that :|  Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/74/commits/0646e20ade6dd091350510081218918da4a1d5d0